utils/doclint/check_public_api/MDBuilder.js

```diff
@@ -59,8 +59,8 @@ class MDOutline {
           let actualText = element.firstChild.textContent;
           let angleIndex = actualText.indexOf('<');
           let spaceIndex = actualText.indexOf(' ');
-          angleIndex = angleIndex === -1 ? angleText.length : angleIndex;
-          spaceIndex = spaceIndex === -1 ? spaceIndex.length : spaceIndex + 1;
+          angleIndex = angleIndex === -1 ? actualText.length : angleIndex;
```

Comment:
What does this fix?

Reply:
There's no such thing as 'angleText'. This fixes a typo.